scripts/variantstore/scripts/import_gvs.py

```diff
@@ -264,7 +264,7 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
             var_ht = hl.import_avro(var_group)
             var_ht = var_ht.transmute(locus=translate_locus(var_ht.location),
                                       local_alleles=hl.array([var_ht.ref]).extend(var_ht.alt.split(',')),
-                                      LGT=hl.parse_call(var_ht.GT),
+                                      LGT=hl.parse_call(hl.or_missing((hl.is_missing(var_ht.GQ) | (var_ht.GQ != 0)), var_ht.GT)),
```

Comment:
Was there a ticket made to address backpatching the Echo VDS GQ 0 genotypes in `merge_and_rescore.py`? I went looking for such a ticket but didn't find one...

Reply:
https://broadworkbench.atlassian.net/browse/VS-1653